### ISSUE TITLE
nersc_hours chokes on jobs with large node counts

### ISSUE THREAD

**Issue by bcfriesen** [NERSC/slurm-helpers#4]
For some large jobs (perhaps those with powers-of-two node counts?), Slurm abbreviates the `nnodes` field:
```
friesen@cori06:~> sacct -j 7799328 -X --format=jobid,elapsed,nnodes
       JobID    Elapsed   NNodes
------------ ---------- --------
7799328        01:33:04       8K
```
The result chokes the `nersc_hours` function:
```
friesen@cori06:~> nersc_hours -knl 7799328
-bash: 8K: value too great for base (error token is "8K")
```

**Comment by bcfriesen** [comment]
It looks like this can be fixed by adding the `--noconvert` flag to `sacct`:
```
friesen@cori06:~> sacct -j 7799328 -X --format=jobid,elapsed,nnodes --noconvert
       JobID    Elapsed   NNodes
------------ ---------- --------
7799328        01:33:04     8192
```

**Comment by sleak-lbl** [comment]
fixed with PR#5
